### ISSUE TITLE
`isFilled.image` does not work with `ImageField` out of the box

### ISSUE THREAD

**Issue by sylvaingi** [prismicio/prismic-helpers#45]
Hello! Seems like a new variation of https://github.com/prismicio/prismic-helpers/issues/41 cropped up in the latest `@prismicio` packages.

### Versions

- @prismicio/helpers: 2.2.0
- @prismicio/types: 0.1.27
- node: v16.14.0
- typescript: 4.6.2

### Steps to reproduce

```ts
type Doc = {
  image: ImageField;
};

const d: Doc = { image: {} };

isFilled.image(d.image);

/*
  Argument of type 'Simplify<(EmptyImageFieldImage | FilledImageFieldImage) & Record<never, EmptyImageFieldImage | FilledImageFieldImage>>' is not assignable to parameter of type 'Simplify<(EmptyImageFieldImage | FilledImageFieldImage) & Record<string, EmptyImageFieldImage | FilledImageFieldImage>> | null | undefined'.
*/
```

### What is expected?

`isFilled.image` shouldn't require a generic argument to be passed in order to typecheck.

### What is actually happening?

Code does not typecheck. `isFilled.image<null>(d.image);` can be used as a workaround.

**Comment by lihbr** [comment]
Hey there, thank you so much for opening an issue again!

The image field type has indeed been giving us some trouble for some time now (as you might have followed on `@prismicio/types` repository) and we hope to see it stabilize.

We'll try to have a look at that quickly!

**Comment by lihbr** [comment]
Released in `2.2.1`, thanks again for your contribution! 🙏
